src/app/[country]/post/page.tsx

```diff
@@ -12,14 +12,15 @@ export default function Page() {
       <div className={country.pageContainer}>
         <div className={country.imageContainer}>
           <Image
+            className={country.image}
             src="/images/countrybg.png"
             alt="countryBackgroundImage"
-            layout="fill"
-            objectFit="cover"
+            fill
+            sizes="1920px"
           />
         </div>
         <div className={country.contentContainer}>
-          <MoveToMain />
+          <MoveToMain country={'중국'} />
```

Comment:
jsx에서 string을 props로 넘길때는 "중국"으로 해도 괜찮습니다. 한쪽으로 통일하시면 좋을 것 같아요

```suggestion
          <MoveToMain country="중국" />
```

Reply:
MoveToMain은 지금은 일단 예시로 '중국'이라고 작성했으나 나중에 변수가 들어갈 자리라서 {} 처리했습니다! 그러나 Image src="" 처럼 변할 여지가 없는 string prop은 모두 {} 없이 string으로 처리해서 수정했습니다 ㅎㅎ!